packages/flutter/test/widgets/nested_scroll_view_test.dart

```diff
@@ -121,6 +121,131 @@ Widget buildTest({
   );
 }
 
+const List<String> tabs = <String>['Hello', 'World'];
+
+MaterialApp buildNestedScrollViewWithTab({
+  required List<String> tabs,
+  void Function()? onBuildHeaderSliver,
+  bool usePageStorageKey = true,
+}) {
+  return MaterialApp(home: Material(child:
+  // THE FOLLOWING SECTION IS FROM THE NestedScrollView DOCUMENTATION
+  // (EXCEPT FOR THE CHANGES TO THE buildCount COUNTER)
```

Comment:
This comment seems outdated. Presumably, the onBuildHeaderSliver is the thing that is not in the original.